bundles/GlossaryBundle/src/Controller/SettingsController.php

```diff
@@ -0,0 +1,145 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\GlossaryBundle\Controller;
+
+
+use Pimcore\Bundle\AdminBundle\Controller\AdminController;
+use Pimcore\Bundle\GlossaryBundle\Model\Glossary;
+use Pimcore\Cache;
+use Pimcore\Model\Document;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/settings")
```

Comment:
I think we cannot remove the admin prefix from the routing, because then we cannot use the AdminController anymore because of the checks. We can simply add the admin back into the name, but I guess we should make it  ```pimcore_admin_glossary_settings_glossary``` then?

Reply:
Ok, I just scanned `Ecommercebundle` and there we use the admin prefix for the route and not in the name so keeping it consistent then we can go without admin prefix in the name. could you please revert it? sorry for rework 😞